packages/react-native-reanimated/android/src/main/cpp/worklets/android/WorkletsModule.cpp

```diff
@@ -52,7 +50,6 @@ jni::local_ref<WorkletsModule::jhybriddata> WorkletsModule::initHybrid(
       std::make_shared<worklets::JSScheduler>(*rnRuntime, jsCallInvoker);
   auto uiScheduler = androidUIScheduler->cthis()->getUIScheduler();
   return makeCxxInstance(
-      jThis,
       rnRuntime,
```

Comment:
This change looks weird, do we really need `WorkletsModule` to be a Hybrid class if we don't use `jThis` here in `makeCxxInstance`?